.github/ISSUE_TEMPLATE/requst-an-example.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Requst an Example
+about: Suggest a new example
+title: ''
+labels: enhancement
+assignees: bisinho, kloyan
+
+---
+
+**Is your example request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
```

Comment:
I would replace "frustrated" with "I am would like to find a solution for this use case / problem I face  when [...]"

Reply:
I'll leave it as it is - this is the default description provided by GitHub